src/TideCoreOperation.php

```diff
@@ -258,4 +258,31 @@ public function enabledTideTfa() {
     }
   }
 
+  /**
+   * Enables paragraphs_library.
+   */
+  public function alterParagraphsLibrary() {
+    // Enabled paragraphs_library module.
+    if (!\Drupal::moduleHandler()->moduleExists('paragraphs_library')) {
+      /** @var \Drupal\Core\Extension\ModuleInstallerInterface $module_installer */
+      $module_installer = \Drupal::service('module_installer');
+      $module_installer->install(['paragraphs_library']);
```

Comment:
I think we should add a helper function to install and uninstall modules with proper try catch error handling. But we can do that separately, doesn't need to be a part of this.